v3/src/components/tool-shelf/tool-shelf.tsx

```diff
@@ -11,7 +17,24 @@ import CalcIcon from '../../assets/icons/icon-calc.svg'
 import TextIcon from '../../assets/icons/icon-text.svg'
 import PluginsIcon from '../../assets/icons/icon-plug.svg'
 
-export const ToolShelf = () => {
+import './tool-shelf.scss'
+
+const kCalcHeight = 162
```

Comment:
Rather than import information from various files, I think it would be better to consolidate all the information for tiles of different types in the registry and pull the information out with getTileComponentInfo.